tools/ot-fct/cli.cpp

```diff
@@ -277,7 +274,7 @@ otError Cli::ProcessCalibrationTable(Utils::CmdLineParser::Arg aArgs[])
 void Cli::ProcessCommand(Utils::CmdLineParser::Arg aArgs[])
 {
     otError error = OT_ERROR_NOT_FOUND;
-    int     i;
+    size_t     i;
```

Comment:
To fix pretty check:
```suggestion
     size_t  i;
```

Reply:
fixed pretty check. 
@jwhui @zhanglongxia , please help to merge PR.